Objects/TI4_Helpers/TI4_FactionHelper.ttslua

```diff
@@ -47,7 +47,8 @@ local _factionAttributes = {
         flagship = 'Duha Menaimon',
         flagshipDescription = 'After you activate this system, you may produce up to 5 units in this system.',
         abilities = { 'Mitosis' },
-        units = { 'Letani Warrior I', 'Letani Warrior II' },
+        units = { 'Letani Warrior I', 'Letani Warrior II', 'Letani Behemoth' },
+        commander = 'Dirzuga Rophal',
```

Comment:
I like adding the commander here, I don't think need agents but can also add hero and use these entries to help "ready cards" not flip them.